packages/lambdas/src/document-bulk-signer.ts

```diff
@@ -14,15 +12,13 @@ const region = getEnvOrFail("AWS_REGION");
 const apiURL = getEnvOrFail("API_URL");
 const bucketName = getEnvOrFail("MEDICAL_DOCUMENTS_BUCKET_NAME");
 
-export const handler = Sentry.AWSLambda.wrapHandler(
-  async (req: DocumentBulkSignerLambdaRequest) => {
-    const { patientId, cxId, requestId } = req;
-    capture.setExtra({ lambdaName: lambdaName });
-    console.log(
-      `Running with envType: ${getEnvType()}, region: ${region}, ` +
-        `bucketName: ${bucketName}, cxId: ${cxId}, patientId: ${patientId}, requestId: ${requestId} `
-    );
+export const handler = Sentry.AWSLambda.wrapHandler(async (req: DocumentBulkSignerRequest) => {
+  const { patientId, cxId, requestId } = req;
+  capture.setExtra({ lambdaName: lambdaName });
+  console.log(
+    `Running with envType: ${getEnvType()}, region: ${region}, ` +
+      `bucketName: ${bucketName}, cxId: ${cxId}, patientId: ${patientId}, requestId: ${requestId} `
+  );
 
-    await getSignedUrls(cxId, patientId, requestId, bucketName, region, apiURL);
-  }
-);
+  await signUrlsAndSendToApi(cxId, patientId, requestId, bucketName, region, apiURL);
```

Comment:
I think that is confusing to use a local factory inside the lambda. The usage of DocumentDownloaderLocal in the lambda caused me a lot of confusion. I see the factories purpose as enabling the  api endpoint to both work during local development and during deployment in cloud env. 
Besides, we wouldnt be able to use the proper pattern of calling the makeDocumentBulkSigner since it would return a DocumentBulkSignerLambda instead of a DocumentBulkSignerLocal, so we wouldnt even be using the pattern properly. 
Unless you feel strongly about this I am going to leave this as is

Reply:
I didn't say anything about using a factory. 😄 

I just suggested to use the exact same code the API uses when being run local, this way we prevent any updates on `DocumentBulkSignerLocal` that would result in diff behavior when testing local vs. running it on the cloud.

Maybe what's confusing is the usage of "Local" as the implementation that directly calls the service instead of going through a lambda. I've thought about that too, maybe a better name would be "Direct"?

Lastly, I didn't understand the point about `makeDocumentBulkSigner`, I don't see it being used in neither of the files.

Happy to huddle.